docs/policy_configuration.md

```diff
@@ -0,0 +1,160 @@
+# Policy Configuration
+
+## Preparation
+TO DO
+
+## Initial install and configuration
+TO DO
+
+## Day 2 configuration
+
+### Updates to ProvisioningRequest spec.templateParameters.clusterInstanceParameters
+
+A ProvisioningRequest can be edited to update:
+* the cluster labels and annotation
+```yaml
+spec:
+  templateParameters:
+    clusterInstanceParameters:
+      clusterName: cnfdg11
+      extraAnnotations:
+        ManagedCluster:
+          test: test <<< added as a Day 2 change
+      extraLabels:
+        ManagedCluster:
+          clustertemplate-a-policy: v1
+          test: test <<< added as a Day 2 change
+```
+* the node labels and annotations
+```yaml
+spec:
```

Comment:
I like it that you included the description, name, templateName in the example below - add it here too?